src/MonthView/MonthView.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react"
+import { Component } from "react"
+import { View } from "react-native"
+import { DayCalendarBox } from "./phone/DayCalendarBox"
+
+interface ComponentProps {}
+
+const weBusy = [
+    {name: "Adam", color: "blue"},
+    {name: "Adi", color: "green"},
+    {name: "Bruno", color: "red"},
+    {name: "George", color: "pink"},
+    {name: "Chiro", color: "black"},
+    {name: "Olivia", color: "grey"}
+]
+
+export class MonthView extends Component<ComponentProps, any> {
```

Comment:
I just realise this is just your placeholder component.

We should name it calendar month or something